BlobStorageContext.cs

```diff
@@ -0,0 +1,59 @@
+using Azure.Storage.Blobs;
+using Azure.Storage.Blobs.Models;
+using Azure.Storage.Blobs.Specialized;
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using System.IO;
+using System.Windows;
+
+
+namespace Wishlist{
+
+    public class BlobStorageContext
+    {
+        private BlobServiceClient blobServiceClient;
+        private const string connStr = @"DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://127.0.0.1:10000/devstoreaccount1;QueueEndpoint=http://127.0.0.1:10001/devstoreaccount1;TableEndpoint=http://127.0.0.1:10002/devstoreaccount1;";
```

Comment:
We try not to version or connection strings since they are private. When they end up in github they could be grabbed by anyone who can access the repo. 

In your case this is just a local string, but when we move to the real azure environment - they will need to be removed. 

To fix this you can set this connection string to a json field in your `local.settings.json` and pull it out using [this method](https://docs.microsoft.com/en-us/dotnet/api/system.environment.getenvironmentvariable?view=net-6.0)

Reply:
Got it, will change it